### DISCUSSION TITLE
Can we enabling the color edging for the window?

### DISCUSSION THREAD

**Discussion by helloyhan** [jshinonome/vscode-q#82] [Ideas]
Hi, it is a great tool for kdb+ in vscode, I'm wondering can we add a new feature that we can enable different colors like for the edges of the window to remind the different servers? e.g. I used to mis-operate some query in the production database whereas it should be run in uat, with the colored edge, it could be a good reminder. I know that the feature is available in QStudio. Thanks!

**Comment by jshinonome** [comment]
Hi, I don't think I can change the color of the text editor(vscode feature), but in the latest version, if you tag server with prod, the status(bottom line) will change to red as below.

![image](https://user-images.githubusercontent.com/64458591/186677244-30b6b9ad-9756-4853-8266-a83642b5e0aa.png)
